FhirToDataLake/deploy/templates/Function/DeployFhirPipelineToFunction.json

```diff
@@ -66,7 +66,28 @@
             "minValue": 2,
             "maxValue": 10,
             "metadata": {
-                "description": "Concurrent jobs executing in parallel."
+                "description": "Count of concurrent jobs executing in parallel."
+            }
+        },
+        "schedulerIntervalInMinutes": {
+            "type": "int",
+            "defaultValue": 5,
+            "allowedValues": [
+            0,
+            5,
+            30,
+            60,
+            1440
+            ],
+            "metadata": {
+                "description": "The scheduler interval in minutes. To customize the scheduler, set the value to 0 and specify the value in schedulerCronExpression below."
```

Comment:
Shall we use a [timespan value](https://blog.ploeh.dk/2019/11/25/timespan-configuration-values-in-net-core/) here? Maybe 0 can be replaced with a special "CronExpression".